src/themes/dark.scss

```diff
@@ -1,10 +1,11 @@
 .#{$WEBEX_COMPONENTS_CLASS_PREFIX}-theme-dark {
-  --wxc-meeting-background: #{$wxc-gradation-webex-dark-primary};
   --wxc-text-color: #{$wxc-white-alpha-95};
   --wxc-secondary-text-color: #{$wxc-white-alpha-70};
   --wxc-warning-color: #{$wxc-yellow-40};
   --wxc-muted-color: #{$wxc-red-50};
   --wxc-unmuted-color: #{$wxc-green-40};
+  --wxc-meeting-background: #{$wxc-gradation-webex-dark-primary};
+  --wxc-member-roster-background: #{$wxc-gradation-webex-dark-secondary};
```

Comment:
we should use --wxc-primary-background and --wxc-secondary-background, to avoid using webex component names in the css variable names